packages/components/tree/src/tree.type.ts

```diff
@@ -63,7 +63,7 @@ export declare interface TreeOptionProps {
   children?: string
   label?: string | ((data: TreeNodeData, node: Node) => string)
   disabled?: string | ((data: TreeNodeData, node: Node) => string)
-  isLeaf?: string | ((data: TreeNodeData, node: Node) => string)
+  isLeaf?: string | ((data: TreeNodeData, node: Node) => boolean)
```

Comment:
I thin `isLeaf` should be `boolean` instead of `string` right? Does `isXx` sounds `string` to you? Because when it is function type it returns `boolean`, when its not function type it should be consistent with `boolean`, IMHO

Reply:
I don't think it's appropriate either but the design is so
<img width="1557" alt="image" src="https://user-images.githubusercontent.com/44761321/153554728-42333edb-f9f4-4ee6-a89f-4f1dd2629c9a.png">